src/state/BoardBoss.js

```diff
@@ -0,0 +1,108 @@
+import { ROWS, COLUMNS } from "../constants";
+
+export const SELECTED = "^";
+
+export const EMPTY = "";
+export const PHAROH = "P";
+export const DJED = "D";
+export const OBELISK = "O";
+export const PYRAMID = "P";
```

Comment:
pyramid and "pharoh" (think it's pharaoh) are both `"P"`. Any reason not to just use the full string, e.g.
```ts
export const PHARAOH = "pharaoh";
export const DJED = "djed";
export const OBELISK = "obelisk";
export const PYRAMID = "pharaoh";
```

Reply:
Nope 😆

Reply:
Also... `export const PYRAMID = "pharaoh";`